content/id/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -0,0 +1,72 @@
+ - Apakah kamu akan menggunakan **Kubernetes kluster di *hosting***, seperti [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/), atau ***hosting* sendiri klustermu**?
+ - Apakah klustermu berada pada ***on-premises***, atau **di cloud (IaaS)**? Kubernetes belum mendukung secara langsung kluster hibrid. Sebagai gantinya, kamu dapat membuat beberapa kluster.
+ - **Jika kamu ingin mengonfigurasi Kubernetes *on-premises***, pertimbangkan [model jaringan](/docs/concepts/cluster-administration/networking/) yang paling sesuai.
+ - Apakah kamu ingin menjalankan Kubernetes pada **"bare metal" *hardware*** atau pada ***virtual machines* (VMs)**?
+ - Apakah kamu **hanya ingin mencoba kluster Kubernetes**, atau kamu ingin ikut aktif melakukan **pengembangan kode dari proyek Kubernetes**? Jika jawabannya yang terakhir, pilihlah distro yang aktif dikembangkan. Beberapa distro hanya menggunakan rilis *binary*, namun menawarkan lebih banyak variasi pilihan.
+ - Pastikan kamu paham dan terbiasa dengan beberapa [komponen](/docs/admin/cluster-components/) yang dibutuhkan untuk menjalankan sebuah kluster.
+
+Catatan: Tidak semua distro aktif dikelola. Pilihlah distro yang telah diuji dengan versi terkini dari Kubernetes.
+
+## Mengelola Kluster
+
+* [Mengelola kluster](/docs/tasks/administer-cluster/cluster-management/) akan menjabarkan beberapa topik terkait *lifecycle* dari kluster: membuat kluster baru, melakukan *upgrade* pada *node master* dan *worker*, melakukan pemeliharaan *node* (contoh: *upgrade* kernel), dan melakukan *upgrade* versi Kubernetes API pada kluster yang sedang berjalan.
+
+* Pelajari bagaimana cara [mengatur *node*](/docs/concepts/nodes/node/).
+
+* Pelajari bagaimana cara membuat dan mengatur [kuota *resource*](/docs/concepts/policy/resource-quotas/) untuk *shared* kluster.
+
+## Mengamankan Kluster
+
+* [Sertifikat (*certificate*)](/docs/concepts/cluster-administration/certificates/) akan menjabarkan langkah-langkah untuk membuat sertifikat menggunakan beberapa *tool chains*.
+
+* [Kubernetes *Container Environment*](/docs/concepts/containers/container-environment-variables/) akan menjelaskan *environment* untuk kontainer yang dikelola oleh Kubelet pada Kubernetes *node*.
+
+* [Mengontrol Akses ke Kubernetes API](/docs/reference/access-authn-authz/controlling-access/) akan menjabarkan bagaimana cara mengatur *permission* untuk pengguna dan *service account*.
+
+* [Autentikasi](/docs/reference/access-authn-authz/authentication/) akan menjelaskan autentikasi di Kubernetes, termasuk ragam pilihan autentikasi.
+
+* [Otorisasi](/docs/reference/access-authn-authz/authorization/) dibedakan dari autentikasi, digunakan untuk mengontrol bagaimana *HTTP call* ditangani.
+
+* [Menggunakan *Admission Controllers*](/docs/reference/access-authn-authz/admission-controllers/) akan menjelaskan *plug-in* yang akan mengintersep permintaan ke server Kubernetes API setelah autentikasi dan otorisasi dilakukan.
```

Comment:
intersep is not indonesian word, perhaps use imported english word or some other Indonesian word?

Reply:
I was looking at kbbi and found https://kbbi.kemdikbud.go.id/entri/intersep. Kompas also using intersep at their articles.
Maybe the intersep is quite unfamiliar for many, so we can keep the imported english word aside intersep like mengintersep (_intercept_)

Reply:
oh ok glad you found this kemdikbud source, we can refer to this aside from

http://kbbi.web.id
http://kbbi.kamus.pelajar.id

please go ahead :+1: